Content.Server/PDA/Ringer/RingerComponent.cs

```diff
@@ -22,7 +22,7 @@ public sealed class RingerComponent : Component
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("range")]
-        public float Range = 3f;
+        public float Range = 0.5f;
```

Comment:
Was this changed for stealth reasons?

Reply:
Yes

Reply:
They can just disable the ringtone though?

Reply:
will do this